### PR TITLE
Review user data in AutoScalingGroup launch configuration

### PR DESCRIPTION
**Pull request by neilkidd**
The user_data was reviewed to ensure no extra data, libs or configs exist.

Two very minor edits made and  logs confirmed in AWS cloudwatch.

### CODE REVIEW

File: modules/dns_dhcp_common/auto_scaling_group.tf [hunk 1]
```diff
@@ -74,7 +74,8 @@ repo_upgrade: all
 MIME-Version: 1.0
 Content-Type: text/x-shellscript; charset="us-ascii"
 #!/bin/bash
-sudo yum install perl-Switch perl-DateTime perl-Sys-Syslog perl-LWP-Protocol-https -y
+# https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/mon-scripts.html
+sudo yum -y install perl-Switch perl-DateTime perl-Sys-Syslog perl-LWP-Protocol-https
```
Comment (emileswarts) [inline modules/dns_dhcp_common/auto_scaling_group.tf:78]: Know you didn't introduce this, but there is no need for sudo in user_data, we can remove these

Comment (neilkidd) [reply modules/dns_dhcp_common/auto_scaling_group.tf]: Thanks - removed in abc713a76fd4f9a2b4080171aaf7413433a288e8

Comment (emileswarts) [reply modules/dns_dhcp_common/auto_scaling_group.tf]: cool, can see a few more sudo's down the file, can we remove those too please?

Comment (neilkidd) [reply modules/dns_dhcp_common/auto_scaling_group.tf]: All removed in 923a128f1091906e148e6f359b70ee16343ea064

**Pull request merged**